documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -102,14 +99,12 @@ Enable or disable 4Kp60 resolution for HDMI outputs.
 
 Enable and disable various physical and virtual interfaces.
 
-image::images/raspi-interface.png[raspi-config interface options]
-
 [[ssh]]
 ==== SSH
 
 Enable or disable remote terminal access to your Raspberry Pi using SSH.
 
-SSH allows you to remotely access the command line of the Raspberry Pi from another computer. SSH is disabled by default. Read more about using SSH on the xref:remote-access.adoc#ssh[SSH documentation page]. If connecting your Raspberry Pi directly to a public network, you should not enable SSH unless you have set up secure passwords for all users.
+SSH allows you to remotely access the command line of the Raspberry Pi from another computer. SSH is disabled by default. If connecting your Raspberry Pi directly to a public network, set up secure passwords for all users before enabling SSH. Learn more about using SSH on the xref:remote-access.adoc#ssh[SSH documentation page].
```

Comment:
```suggestion
SSH allows you to remotely access the command line of the Raspberry Pi from another computer. SSH is disabled by default. If connecting your Raspberry Pi directly to a public network, set up passwords for all users before enabling SSH. Learn more about using SSH on the xref:remote-access.adoc#ssh[SSH documentation page].
```

While users absolutely should use a secure password, I think it goes without saying. Let's trim it here.

Reply:
ping @tdewey-rpi for his thoughts here, just to be on the safe side. I guess at the very least we ought to link to https://www.raspberrypi.com/documentation/computers/configuration.html#secure-your-raspberry-pi ?

Reply:
You summoned me, @lurch?

I agree with both of you.

1. I don't think we need to say 'secure password' - what does that even mean, and how can you be sure something that is 'secure' today is considered 'secure' tomorrow, etc.
2. We can, however, provide a link to the securing page - and then change this to a later security centre.

Reply:
:wave: 

I've just re-read Nate's actual suggested wording - "If connecting your Raspberry Pi directly to a public network, set up passwords for all users before enabling SSH." - AFAIK it's not possible to have a user on Raspberry Pi OS _without_ a password... :thinking:

Reply:
In that case, we can probably just remove the sentence entirely!

I think I have managed to create a user without a password before. But it may have been an older version of the OS.